plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/scanner/ClassPathBeanRefreshCommand.java

```diff
@@ -48,22 +52,24 @@ public class ClassPathBeanRefreshCommand extends MergeableCommand {
     WatchFileEvent event;
     byte[] classDefinition;
 
-    public ClassPathBeanRefreshCommand() {
+    Scheduler scheduler;
 
-    }
-
-    public ClassPathBeanRefreshCommand(ClassLoader appClassLoader, String basePackage, String className, byte[] classDefinition) {
+    public ClassPathBeanRefreshCommand(ClassLoader appClassLoader, String basePackage, String className,
+                                       byte[] classDefinition, Scheduler scheduler) {
         this.appClassLoader = appClassLoader;
         this.basePackage = basePackage;
         this.className = className;
         this.classDefinition = classDefinition;
+        this.scheduler = scheduler;
     }
 
-    public ClassPathBeanRefreshCommand(ClassLoader appClassLoader, String basePackage, String className, WatchFileEvent event) {
+    public ClassPathBeanRefreshCommand(ClassLoader appClassLoader, String basePackage, String className,
```

Comment:
this looks not right to me, a scheduled command will schedule another command. I suggest simply call ClassPathBeanDefinitionScannerAgent's refreshClassAndCheckReload method and then schedule SpringChangedReloadCommand in where ClassPathBeanRefreshCommand is inited. 

In other word, this command is unnecessary to exist any longer.